frontend/src/types/Ministry.d.ts

```diff
@@ -0,0 +1,8 @@
+import { InquiryData } from ".";
+
+export type MinistryData = {
+    ministry_id: string;
+    ministry_name: string;
+    inquiries: InquiryData[];
+  };
+  
```

Comment:
Just a little line ending issue here that needs fixing :)

Reply:
> Just a little line ending issue here that needs fixing :)

thanks for checking it! I removed the space on the ending part.